libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -104,32 +104,31 @@ export const TicketReplyForm: React.FC<{
               maxSize={3145728}
             />
 
-            <div className="status-wrapper">
-              <Label>Ticket Status</Label>
-              {ticketData?.Status !== 'resolved' ? (
-                <div className="status-checkbox">
-                  <Field type="checkbox" name="status" />
-                  My issue has been resolved
-                </div>
-              ) : (
-                <div className="status-checkbox">
-                  <Field type="checkbox" name="status" checked disabled /> My
-                  issue has been resolved
-                </div>
-              )}
-              {ticketData?.Status === 'resolved' ? (
-                <em>*Replying will reopen this ticket</em>
-              ) : (
-                <em>
-                  This helps us determine which users still need assistance
-                </em>
-              )}
+            <div className="c-form__field">
+              <label>Ticket Status</label>
+              <menu>
+                <li>
+                  {ticketData?.Status !== 'resolved' ? (
+                  <label>
+                    <Field type="checkbox" name="status" />
+                    My issue has been resolved
+                  </label>
+                  ) : (
+                  <label>
+                    <Field type="checkbox" name="status" checked disabled />
+                    My issue has been resolved
+                  </label>
+                  )}
```

Comment:
**Redundant Markup.** Can this be less redundant? Idea first proposed in https://github.com/TACC/tup-ui/pull/404/files#r1447971645.